qcfractal/queue/managers.py

```diff
@@ -469,6 +531,23 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
         # Get new tasks
         payload = self._payload_template()
         payload["data"]["limit"] = open_slots
+
+        # Crunch Statistics
+        self.statistics.total_failed_tasks += n_fail
+        self.statistics.total_successful_tasks += n_success
+        self.statistics.total_cpu_hours += task_cpu_hours
+        self.logger.info(f"Stats: Number of processed jobs: {self.statistics.total_completed_tasks}")
+        self.logger.info(f"Stats: {self.statistics.total_successful_tasks} successful / "
+                         f"{self.statistics.total_failed_tasks} failed "
+                         f"({self.statistics.total_successful_tasks/self.statistics.total_completed_tasks*100}% "
+                         f"success rate)")
+        self.logger.info(f"Stats: {self.statistics.total_cpu_hours} CPU Hours logged successfully (estimate)")
```

Comment:
Can we try to make this a single line?

`(processed=5, failed=1, success=80%, core_hours=53.3)`

Reply:
Yes. I like that better